agilent_infiniium_scope.py

```diff
@@ -72,319 +72,350 @@ def channel_infoNONEXISTANT(self, c, channel):
         Tuple of (probeAtten, termination, scale, position, coupling, bwLimit, invert, units)
         """
         raise Exception('Not yet implemented')
-        #NOTES
-        #The scope's response to 'CH<x>?' is a string of format
-        #'1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
-        #These strings represent respectively,
-        #probeAttenuation;termination;vertScale;vertPosition;coupling;bandwidthLimit;invert;vertUnit
+        """
+        NOTES
+        The scope's response to 'CH<x>?' is a string of format
+        '1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
+        These strings represent respectively,
+        probeAttenuation;termination;vertScale;vertPosition;coupling;
+        bandwidthLimit;invert;vertUnit
+        """
 
         dev = self.selectedDevice(c)
-        resp = yield dev.query('CH%d?' %channel)
-        bwLimit, coupling, deskew, offset, invert, position, scale, termination, probeCal, probeAtten, resistance, unit, textID, textSN, extAtten, extUnits, textLabel, xPos, yPos = resp.split(';')
-
-        #Convert strings to numerical data when appropriate
-        probeAtten = T.Value(float(probeAtten),'')
-        termination = T.Value(float(termination),'')
-        scale = T.Value(float(scale),'')
-        position = T.Value(float(position),'')
+        resp = yield dev.query('CH{}?'.format(channel))
+        (bwLimit, coupling, deskew, offset, invert, position, scale,
+         termination, probeCal, probeAtten, resistance, unit, textID,
+         textSN, extAtten, extUnits, textLabel, xPos, yPos) = resp.split(';')
```

Comment:
Ugh, that's gross. I guess this formatting is all a bit academic given that the function throws an exception on the first line, though...